linux/jdk/redhat/src/packageTest/java/packaging/RedHatFlavoursWithYum.java

```diff
@@ -34,6 +34,9 @@
 public class RedHatFlavoursWithYum implements ArgumentsProvider {
 	@Override
 	public Stream<? extends Arguments> provideArguments(ExtensionContext context) {
+		String arch = System.getProperty("os.arch");
+		boolean ubi7_supported = "x86-64".equalsIgnoreCase(arch) || "amd64".equalsIgnoreCase(arch) || "ppc64le".equalsIgnoreCase(arch) || "s390x".equalsIgnoreCase(arch);
```

Comment:
Can someone confirm that this is a correct value of `System.getProperty("os.arch")`, and not "x86_64"? I wasn't sure which of these to put based on what I found on the internet

Reply:
In Java, System.getProperty("os.arch") returns a string representing the underlying system's architecture. For machines running on x86 64-bit architectures, the typical return value is "amd64" or "x86_64" depending on the specific JVM implementation. However, "amd64" is the more common return value. Note that the exact return value may vary between different JVM vendors and versions.

Reply:
I just pushed a new change to fix this. It gets rid of non-alphanumeric characters before comparing the arch to predetermined values